### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by shuvashish76** [MuntashirAkon/AppManager#1106]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [ ] I know how to reproduce the issue which may not be specific to my device

### Describe the bug
App crashed while updating an app from Aurora store. Intermittent crash.

### Logs

```
java.lang.NoSuchFieldError: No field installerPackageName of type Ljava/lang/String; in class Landroid/content/pm/PackageInstaller$SessionParams; or its superclasses (declaration of 'android.content.pm.PackageInstaller$SessionParams' appears in /system/framework/framework.jar)
    at io.github.muntashirakon.AppManager.apk.installer.PackageInstallerCompat.openSession(PackageInstallerCompat.java:756)
    at io.github.muntashirakon.AppManager.apk.installer.PackageInstallerCompat.install(PackageInstallerCompat.java:591)
    at io.github.muntashirakon.AppManager.apk.installer.PackageInstallerService.onHandleIntent(PackageInstallerService.java:171)
    at io.github.muntashirakon.AppManager.types.ForegroundService$ServiceHandler.handleMessage(ForegroundService.java:54)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loop(Looper.java:164)
    at android.os.HandlerThread.run(HandlerThread.java:65)
```
### Device info

App version: 3.2.0-alpha01-DEBUG
App version code: 433
Android build version: 8704X_S001025_190606_ROW
Android release version: 8.1.0
Android SDK version: 27
Android build ID: TB-8704X_S001025_190606_ROW
Device brand: Lenovo
Device manufacturer: LENOVO
Device name: TB-8704X
Device model: Lenovo TB-8704X
Device product name: TB-8704X
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-IN
In-App Language: auto
Mode: auto
Inferred Mode: No root

**Comment by shuvashish76** [comment]
Sorry browser refreshed & duplicate created due to some internet issues. Issue title & description changed.

**Comment by MuntashirAkon** [comment]
Fixed in dba3e90ba2238bd8319d2eeed0cf7b6ded099822
